draft-ietf-quic-ack-frequency.md

```diff
@@ -451,6 +451,11 @@ will wait 'max_ack_delay', increasing the chances of a premature PTO.
 Therefore, if Ignore Order is enabled, the PTO MUST be larger than the peer's
 'max_ack_delay'.
 
+When sending PTO packets, one can either skip a sufficient number of packet numbers
```

Comment:
Why would you rather skip packet numbers than using immediate_ack? I would say immediate_ack is the more explicit solution and we should only recommend that.

Reply:
Fair point, updated.